approach.md

```diff
@@ -0,0 +1,60 @@
+# OSEP proposed approach
+
+a) Document scope of analysis using STPA
+
+*In collaboration with other WGs e.g. Automotive?*
+
+* Assumed system context
+    - Hardware features, role of OS, boundaries of analysis, etc
+    - Doesn't have to be concrete / complex
+      - Specific to topic: start simple and elaborate later!
+* Losses
+    - OS-related outcomes that *may* violate a system's safety goals
+      - i.e. lead to harm in a safety-related system
+* Hazards
+    - OS-level system conditions that *may* lead to these losses
+* System-level constraints
```

Comment:
1. Agreed. The system-level constraints represent the top-level safety requirements / safety goals. I will add this.
2. The intended purpose of using STPA is to derive more detailed safety requirements from these high level constraints, by defining controller constraints (which specify the criteria for avoiding UCAs) and Loss Scenarios (which can be used in combination with Controller Constraints to define test cases, including fault injection test cases)

Reply:
WRT to point 2 I understand the goal of STPA as described in the handbook however if we want to use an iterative approach we need to define new constraints and new controllers at each iteration inside the Kernel. In doing so I find it a bit  confusing to use the term "constraints" instead of "SW Safety Requirements and AoUs"....